daphne/src/lib.rs

```diff
@@ -1118,6 +1118,10 @@ pub struct DapRequest<S> {
     /// requests may apply to all tasks, e.g., the request for the HPKE configuration.
     pub task_id: Option<TaskId>,
 
+    /// ID of the collection job. This field is optional, since some
+    /// requests may apply to all tasks.
```

Comment:
What do you mean by "some requests may apply to all tasks"?

In what situations would the collection job ID not be part of the DAP request?